src/main/webapp/assets-lti/js/factories/factories.js

```diff
@@ -114,8 +114,15 @@ canvasSupportApp.factory('Course', function ($http) {
 //SECTIONS FACTORY - does the request for the sections controller
 canvasSupportApp.factory('Sections', function ($http) {
   return {
-    getSectionsForCourseId: function (courseId) {
-      var url = '/canvasCourseManager/manager/api/v1/courses/' + courseId + '/sections?per_page=100&_='+ generateCurrentTimestamp();
+    getSectionsForCourseId: function (courseId, context) {
+    	console.log('Course: ' + courseId);
+    	console.log('Context: ' + context);
+    	if ( context === true){
```

Comment:
How about using the context === true test just to set the correct value for a new course id variable and then just building the url once using that variable?  It avoids a lot of repeated code and makes it clearer what is different.